### DISCUSSION TITLE
CVE-2023-2975 and FIPS module (openssl 3.0.9)

### DISCUSSION THREAD

**Discussion by joelguittet** [openssl/openssl#24702] [Q&A] [answered]
Hello,

The description of the CVE-2023-2975 (https://www.openssl.org/news/vulnerabilities-3.1.html) doesn't indicate anything about the FIPS module. This CVE is related to AES-SIV algorithm and has been fixed with a modification of cipher_aes_siv.c.

However, the AES-xxx-SIV algorithm are indicated as NOT available in the FIPS module here: https://www.openssl.org/docs/man3.3/man7/EVP_CIPHER-AES.html

So I conclude the FIPS module is not vulnerable to the CVE-2023-2975 (only the default legacy provider is vulnerable). Is it correct ?

Thanks!
Joel

**Comment by nhorman** [comment]
@joelguittet I believe the answer to your question is that you are correct, the FIPS module is not impacted.

@t8m can you confirm please?

**Reply by joelguittet** [reply]
@nhorman thanks for the quick answer to this question!

So I guess the description of the CVE can be updated with "The OpenSSL 3.0 and 3.1 FIPS providers are not affected by this issue." ? :) Who manage this ?

@t8m I'm waiting your confirmation with great interest, thanks!

**Reply by t8m** [reply, marked as answer]
@joelguittet please refer to this page for the relevance of CVEs to FIPS providers https://www.openssl.org/news/fips-cve.html

**Reply by joelguittet** [reply]
Oh perfect, I have not found this page before and this confirm my conclusions! Thanks!
Topic is closed :-)
